packages/block-library/src/gallery/block.json

```diff
@@ -61,10 +61,10 @@
 		},
 		"shortCodeTransforms": {
 			"type": "array",
-			"default": [],
 			"items": {
 				"type": "object"
-			}
+			},
+			"default": []
```

Comment:
This is just an aesthetic drive-by change. The intent is to consistently declare the `default` as the last field. See, for example, the `ids` definition that precedes the `shortCodeTransforms` one.